dev/storage_base.h

```diff
@@ -846,8 +871,8 @@ namespace sqlite_orm {
             std::map<std::string, collating_function> collatingFunctions;
             const int cachedForeignKeysCount;
             std::function<int(int)> _busy_handler;
-            std::vector<std::unique_ptr<udf_proxy_base>> scalarFunctions;
-            std::vector<std::unique_ptr<udf_proxy_base>> aggregateFunctions;
+            std::vector<std::unique_ptr<udf_proxy>> scalarFunctions;
```

Comment:
@fnc12 Since `udf_proxy` is now a complete structure itself (without derived classes), do you mind using an `std::list<udf_proxy>` instead of the `unique_ptr` indirection?

Reply:
yes it is a good call